### PR TITLE
My Workflows

### PR DESCRIPTION
**Pull request by garyluu**
For frontend part of dockstore/dockstore#3434

The CollectionEntry changes surprisingly isn't typed here so it doesn't fail (not a good thing).  
MyWorkflows type has been left unchanged so there's no changes here.

I tested this locally, but there will need to be another PR to update the package.json to 1.8.6 once that's released (and not use the CircleCI jar).

### CODE REVIEW

File: scripts/generate-openapi-script.sh [hunk 1]
```diff
@@ -8,11 +8,14 @@ GENERATOR_VERSION="3.3.4"
 BASE_PATH="https://raw.githubusercontent.com/dockstore/dockstore/$npm_package_config_webservice_version"
 # DOCKSTORE-2428 - demo how to add new workflow language, generate from local copy of swagger
 # BASE_PATH="../dockstore"
+#CIRCLE_CI_PATH="https://3855-33383826-gh.circle-artifacts.com/0/tmp/artifacts"
```
Comment (denis-yuen) [inline scripts/generate-openapi-script.sh:11]: Intentionally left in? 
If so, might need to code comment `3855-33383826` (is that unique to "our" repo on circle?)

File: scripts/run-webservice-script.sh [hunk 2]
```diff
@@ -4,7 +4,10 @@ set -o pipefail
 set -o nounset
 set -o xtrace
 
-wget -O dockstore-webservice.jar --no-verbose --tries=10 https://artifacts.oicr.on.ca/artifactory/collab-release/io/dockstore/dockstore-webservice/${npm_package_config_webservice_version}/dockstore-webservice-${npm_package_config_webservice_version}.jar
+#JAR_PATH="https://artifacts.oicr.on.ca/artifactory/collab-release/io/dockstore/dockstore-webservice/${npm_package_config_webservice_version}/dockstore-webservice-${npm_package_config_webservice_version}.jar"
+JAR_PATH="https://3855-33383826-gh.circle-artifacts.com/0/tmp/artifacts/dockstore-webservice-1.8.6-SNAPSHOT.jar"
```
Comment (denis-yuen) [inline scripts/run-webservice-script.sh:8]: Ditto

File: scripts/generate-openapi-script.sh (hunk 1 above)
Comment (garyluu) [reply scripts/generate-openapi-script.sh:11]: It's intentionally left in for when I or others need to use a JAR for CircleCI.  The URL is apparent when you already uploaded a JAR via CircleCI

File: scripts/run-webservice-script.sh (hunk 2 above)
Comment (garyluu) [reply scripts/run-webservice-script.sh:8]: This is going to be replaced by the next 1.8.6 package.json PR

File: scripts/generate-openapi-script.sh (hunk 1 above)
Comment (denis-yuen) [reply scripts/generate-openapi-script.sh:11]: Sounds good, add that as a code comment linking to https://wiki.oicr.on.ca/display/DOC/Dockstore+Releases#DockstoreReleases-PreDeploy and add a screenshot of where to look in CircleCI, thanks!

**Pull request merged**